spi/common/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/query/QuerySpec.java

```diff
@@ -29,6 +29,12 @@
  * is tunnelled through to the database level.
  */
 public class QuerySpec {
+
+    public static final String QUERY_SPEC = "querySpec";
+    public static final String FILTER_EXPRESSION = "filterExpression";
```

Comment:
These constants (except `QUERY_SPEC`) are only used in the `DescriptionRequestHandlerTest`. Please move them out of the production code base.

Reply:
Done.